jenkins/check_copyright.py

```diff
@@ -42,6 +42,8 @@ def report_incorrect(file_name, pairs):
 linecount = 0
 pairs = []
 with open(file_name, 'r') as sourcefile:
+    if not re.match(r'#!/usr/bin/env.*', sourcefile.readline(), 0):
```

Comment:
Why not just `sourcefile.readline().startswith("#!")`? I think we're imposing too much structure on the shebang.